pkg/shell/hosts.jsx

```diff
@@ -197,7 +197,15 @@ export class CockpitHosts extends React.Component {
                     { this.state.opened &&
                     <HostsSelector>
                         <PageSidebar isSidebarOpen={this.props.opened} theme="dark" className={"sidebar-hosts" + (this.state.editing ? " edit-hosts" : "")}>
-                            <CockpitNav selector={this.props.selector} groups={groups} item_render={render} sorting={(a, b) => true} filtering={this.filterHosts} current={label} />
+                            <CockpitNav
+                                selector={this.props.selector}
+                                groups={groups}
+                                item_render={render}
+                                sorting={(a, b) => true}
+                                filtering={this.filterHosts}
+                                current={label}
+                                jump={() => console.error("internal error: jump not supported in hosts selector")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19503-20231018-115234-5f9f5ce3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)